setup-schema.json

```diff
@@ -0,0 +1,24 @@
+{
+    "$schema": "http://json-schema.org",
+    "$id": "http://yio-remote.com/yioroon.json",
+    "type": "object",
+    "title": "YIO Roon Integration Schema",
+    "description": "Required data points to set up a YIO Roon integration.",
+    "default": {},
+    "additionalProperties": true,
+    "required": [
+        "ip"
+    ],
+    "properties": {
+        "ip": {
+            "$id": "#/properties/ip",
+            "type": "string",
+            "title": "IP address or hostname",
+            "description": "The IP address or hostname of your Roon Core.",
```

Comment:
The integration checks if the port is given and appends it automatically. I think the title and description should say that port can be given as optional.

Reply:
I am wondering if the port should always be added. I am not aware that Roon allows you to change the port...

Reply:
Anyway, don't want to change too much. Will add that.

Reply:
I don't know much about roon, just looking at the integration code that it allows for it.